android-components/components/browser/toolbar/src/main/res/values-ldrtl/dimens.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- DisplayToolbar -->
+    <dimen name="mozac_browser_toolbar_origin_padding_end">16dp</dimen>
```

Comment:
Just for my own curiosity, I am wondering how we came up with 16dp?

Reply:
Sure, it looked consistent with the other/icon paddings and at the start/end. Happy to update if there's a specific grid we adhere to.

Reply:
No worries, I think I just need a moment later today to pull the down the change and try to rationalize the padding of everything and see if the value fits with the existing layout. Normally, we would ask UX to double check if the screenshots look right, that said visually it looks right to me.

Reply:
Sounds good! 👍